src/utils.ts

```diff
@@ -189,10 +189,16 @@ export function cacheFile(
   }
 }
 
-export function injectSourcemapHook(src: string): string {
+export function injectSourcemapHook(
+  filePath: string,
+  typeScriptCode: string,
+  src: string,
```

Comment:
The `typeScriptCode` and `src` are related but different. I do need to understand what/why we need babel after TypeScript (later).